### DISCUSSION TITLE
Button "Go to beatmap" on daily map in daily challenge menu should direct to mapset ?

### DISCUSSION THREAD

**Discussion by Glebsin** [ppy/osu#30398] [Q&A]
https://github.com/user-attachments/assets/71a90402-57c5-495c-8218-a1f8bfec6e4c

**Comment by peppy** [comment]
The reason it doesn't is because it sees that the beatmap is already selected and therefore decides there's no necessary action.

This was an intentional change to fix an issue: https://github.com/ppy/osu/commit/3c05b975a08dacc644f9877f49f58e40bf92668b

Not sure how we can do this better.
